mlrun/api/utils/scheduler.py

```diff
@@ -367,8 +394,29 @@ async def submit_run_wrapper(scheduled_object, schedule_name):
         # otherwise all runs will have the same uid
         scheduled_object.get("task", {}).get("metadata", {}).pop("uid", None)
 
+        schedule_name_label = "schedule-name"
```

Comment:
Change to `mlrun/schedule-name`
Also please create class `LabelNames` in `mlrun/api/schemas/constants.py` and use it from there